apps/schema/static/schema/components/devices/deviceTelemetry.yaml

```diff
@@ -0,0 +1,41 @@
+post:
+  summary: Add telemetry data
+  description: |
+    Associate telemetry data with a `Shipment`. The telemetry payload must be signed and encoded as a [JWS](https://tools.ietf.org/html/rfc7515).
+    An active AXLE certificate must be used to sign this JWS, and the certificate ID must be included in the headers as `kid`.
+
+    A JSON Schema specification for the telemetry data payload format is located at [schema.shipchain.io](http://schema.shipchain.io/1.2.4/telemetry.json)
+
+    An example of a raw payload:
+    ```
+    {
+      "version": "1.0.1",
+      "timestamp": "2018-08-22T17:44:39.874352",
+      "device_id": "1243d23b-e2fc-475a-8290-0e4f53479553"
+      "sensor_id": "1243d23b-e2fc-475a-8290-0e4f53479553"
+      "hardware_id": "1243d23b-e2fc-475a-8290-0e4f53479553"
```

Comment:
Hardware ID is not necessarily a UUID

Reply:
Yes, but I figured a uuid would be adequate for describing it in swagger as it will be unique formatting per type. What do you think would be a good example for the hardware_id?